project_euler/README.md

```diff
@@ -114,5 +114,5 @@ def solution(arg1: [type hint], arg2: [type hint], ...) -> [Return type hint]:
 
 
 if __name__ == "__main__":
-    print(f"{solution() = }")
+    print(f"solution() = {solution()}")
```

Comment:
You must be running on a Python < 3.8 where this feature was added.  Please upgrade.

Reply:
Ok, sorry for not checking it first